collection-scripts/gather

```diff
@@ -78,38 +76,26 @@ case "$CLUSTER" in
     "HUB")
 
     check_managed_clusters
-    #oc adm inspect  ns/open-cluster-management  --dest-dir=must-gather
-    oc get pods -n "$HUBNAMESPACE" > ${BASE_COLLECTION_PATH}/gather-acm.log
-    oc get csv -n "$HUBNAMESPACE" > ${BASE_COLLECTION_PATH}/gather-acm.log 
-    oc adm inspect  ns/"$HUBNAMESPACE"  --dest-dir=must-gather
-    oc adm inspect  ns/open-cluster-management-hub  --dest-dir=must-gather
+    oc get pods -n "backplane-operator-system" > ${BASE_COLLECTION_PATH}/gather-acm.log
```

Comment:
Since a customer can choose the namespace to deploy in this namespace shouldn't be hardcoded and should use the namespace variable